src/__snapshots__/core.test.ts.snap

```diff
@@ -1680,7 +1680,7 @@ exports[`Core printSummaryOfResults 1`] = `
 ╟─────────────────────────────┼───────┼─────────┼────────────┼─────────────────╢
 ║ ]8;;https://eslint.org/docs/rules/semisemi]8;;                        │ [31m[1m14[39m[22m    │ [31m[1m1[39m[22m       │ [31m[1m15[39m[22m         │ 0               ║
 ╟─────────────────────────────┼───────┼─────────┼────────────┼─────────────────╢
-║ ]8;;https://github.com/import-js/eslint-plugin-import/blob/v2.26.0/docs/rules/order.mdimport/order]8;;                │ [31m[1m3[39m[22m     │ 0       │ [31m[1m3[39m[22m          │ 0               ║
+║ ]8;;https://github.com/import-js/eslint-plugin-import/blob/v2.27.5/docs/rules/order.mdimport/order]8;;                │ [31m[1m3[39m[22m     │ 0       │ [31m[1m3[39m[22m          │ 0               ║
```

Comment:
Since this PR does not update the package version, it is wrong to update this snapshot.

I think this test is failing because you have not installed the version of the package based on pnpm-lock.yaml. Are you installing the package with `npm install` or `yarn install` instead of `pnpm instaii`?

Reply:
I used npm, not pnpm, so that explains it.  Thanks for fixing.